realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -593,48 +569,6 @@ private boolean isPrimaryKeyColumn(long columnIndex) {
     }
 
     /**
-     * Returns a view sorted by the specified column and order.
-     *
-     * @param columnIndex the column index.
-     * @param sortOrder the sort order.
-     * @return a sorted view.
-     */
-    public TableView getSortedView(long columnIndex, Sort sortOrder){
```

Comment:
Does this influence my use-case here

    return resultPost.getResultList().where().findAll().sort(ResultListFields.RANK Sort.ASCENDING);

?